filewatch/followers.go

```diff
@@ -130,6 +130,39 @@ func (f *follower) FileId() FileId {
 	return f.id
 }
 
+func (f *follower) Sync(qc chan os.Signal) (bool, error) {
+	f.mtx.Lock()
+	defer f.mtx.Unlock()
+	if f.lnr == nil {
+		return false, ErrNotReady
+	}
+	if f.abortCh != nil || f.running != 0 {
+		return false, ErrAlreadyStarted
+	}
+	for {
```

Comment:
This code duplicates a lot of processLines, can we just use that or is there special sauce?

Reply:
the quit channel is kind of unique, i made the judgement call that the duplication was better than trying to handle having a quitchan on process lines